test/shared/src/main/scala/zio/test/Gen.scala

```diff
@@ -221,6 +229,19 @@ object Gen extends GenZIO with FunctionVariants with TimeVariants {
   val anyUnicodeChar: Gen[Random, Char] =
     Gen.oneOf(Gen.char('\u0000', '\uD7FF'), Gen.char('\uE000', '\uFFFD'))
 
+  /**
+   * A generator of universally unique identifiers. The returned generator will
+   * not have any shrinking.
+   */
+  val anyUUID: Gen[Random, UUID] =
+    for {
+      mostSigBits  <- Gen.anyLong.noShrink
+      leastSigBits <- Gen.anyLong.noShrink
+    } yield new UUID(
+      (mostSigBits & ~0x0000f000) | 0x00004000,
+      (leastSigBits & ~(0xc0000000.toLong << 32)) | (0x80000000.toLong << 32)
```

Comment:
You can avoid the conversion here
```suggestion
      (leastSigBits & ~(0xc0000000L << 32)) | (0x80000000L << 32)
```

Reply:
Great suggestion!